localstack/utils/bootstrap.py

```diff
@@ -1116,7 +1116,7 @@ def configure_container(container: Container):
     else:
         container.config.additional_flags = f"{container.config.additional_flags} {user_flags}"
 
-    # get additional parameters from plugins
+    # get additional parameters from pluxs
```

Comment:
Looks like a search-replace issue here 😄 
```suggestion
    # get additional parameters from plux
```